src/core/cgroup.c

```diff
@@ -1019,7 +1019,8 @@ static int lookup_block_device(const char *p, dev_t *ret) {
         }
 
         /* If this is a LUKS/DM device, recursively try to get the originating block device */
-        while (block_get_originating(*ret, ret) > 0);
+        while (block_get_originating(*ret, ret) >= 0)
```

Comment:
urks... it used to return > 0, this was broken in 612fc70fc0f5445817f3f5c033dd3d5b5fd058ea

i wonder how nobody noticed that? this is really weird.

Reply:
oh, hmm, i guess, we generally only have one level of "slaves", hence noone noticed.

but anyway, please augment the commit msg with a reference to that commit

Reply:
block_device_get_originating() retains the logic of returning > 0 btw, maybe that should be streamlined...